lib/galaxy/util/__init__.py

```diff
@@ -721,7 +721,7 @@ def __init__(self, params, sanitize=True):
                     key not in self.NEVER_SANITIZE and
                     True not in [key.endswith("|%s" % nonsanitize_parameter) for
                                  nonsanitize_parameter in self.NEVER_SANITIZE]):
```

Comment:
See https://www.python.org/dev/peps/pep-0008/#indentation : "When the conditional part of an `if`-statement is long enough to require that it be written across multiple lines, it's worth noting that the combination of a two character keyword (i.e. `if`), plus a single space, plus an opening parenthesis creates a natural 4-space indent for the subsequent lines of the multiline conditional. This can produce a visual conflict with the indented suite of code nested inside the `if`-statement, which would also naturally be indented to 4 spaces. This PEP takes no explicit position on how (or whether) to further visually distinguish such conditional lines from the nested suite inside the if-statement."
Examples follow there, I think the 3rd one is the best option to resolve the above problem, but it's not enforced.